conf/default-vars.yaml

```diff
@@ -0,0 +1,35 @@
+vars:
+  #families
+  font-primary: 'sans-serif'
+  font-secondary: 'serif'
+  font-heading: 'serif'
+
+  #sizes
+  font-size-body: 14px
+  font-size-large: 16px
+  font-size-small: 12px
+
+  line-height: 1.1
+  line-height-large: 1.5
+
+  #weights
+  font-light: 300
+  font-regular: 400
+  font-semibold: 600
+  font-bold: 700
+  font-xbold: 800
+
+  #headings
+  heading-line-height: 120%
+  heading-1-size: 45px
+  heading-1-mobile-size: 30px
+  heading-2-size: 30px
+  heading-2-mobile-size: 25px
+  heading-3-size: 25px
+  heading-3-mobile-size: 20px
+  heading-4-size: 16px
+  heading-4-mobile-size: 14px
+  heading-5-size: 14px
+  heading-5-mobile-size: 14px
+
+  stretch-spacing: 2px
```

Comment:
True.  But `stretch-spacing` feels like it could be used for more than it is, which we may want but probably not.